module/src/components/tagInput/tagInput.component.tsx

```diff
@@ -197,6 +200,7 @@ export const TagInput = React.forwardRef<HTMLInputElement, ITagInputProps>(
       <>
         {currentTags?.map((tag, index) => (
           <Tag
+            {...tag.htmlProps}
```

Comment:
I think you'll also wanna concat that with the className, like

```tsx
className={ClassNames.concat('arm-tag-input-tag or whatever it is', tag.htmlProps.className)}
```

Reply:
Wasn't sure about this one as in `tag.component.tsx` this is happening
```
export const Tag: React.FC<ITagProps> = ({ content, className, leftIcon, rightIcon, children, onRemove, ...nativeProps }) => (
  <div className={ClassNames.concat('arm-tag', className)} {...nativeProps}>
  ```
  
  which already seems to be taking care of it?

Reply:
Ah wait yeah if the tag input doesn't actually add its own classname to the tag then ignore me